po/ru.po

```diff
@@ -1387,7 +1387,7 @@ msgstr "Перевозчик"
 
 #: pkg/kubernetes/scripts/volumes.js:205
 msgid "Ceph Filesystem Mount"
-msgstr "Крепление файловой системы Ceph"
+msgstr "Монтирование файловой системы Ceph"
```

Comment:
Reference: https://ru.wikipedia.org/wiki/Mount uses that word as well, although it's not on the [leo dict](https://dict.leo.org/russisch-deutsch/%D0%9C%D0%BE%D0%BD%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5).